airbyte-cdk/python/airbyte_cdk/sources/declarative/parsers/class_types_registry.py

```diff
@@ -10,7 +10,7 @@
     ApiKeyAuthenticator,
     BasicHttpAuthenticator,
     BearerAuthenticator,
-    SessionTokenAuthenticator,
+    LegacySessionTokenAuthenticator,
```

Comment:
@girarda It seems that when was changed the `SessionTokenAuthenticator` to `LegacySessionTokenAuthenticator`, wasn't changed this file, so `mypy` complained about this one. Let me know if this is not the case

Reply:
interesting. would you mind fixing this in a separate PR? no reason to delay this fix with the stripe partition work

Reply:
sure, I've created separate [PR](https://github.com/airbytehq/airbyte/pull/32040) fir this